Tests/Unit/Controller/Admin/Transaction/TransactionTabPostProcessingTest.php

```diff
@@ -75,6 +77,23 @@ public function testRenderBasic()
         $this->assertArrayHasKey('emptyText', $this->_transactionTabPostProcessing->getViewData());
     }
 
+    public function testRenderSepaCredit()
+    {
+        $oPayment = oxNew(\Wirecard\Oxid\Extend\Model\Payment::class);
```

Comment:
Please import these classes via a `use` statement at the top like we do for all other classes.